_CodingInTheCabana/001-maurer-rose.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Maurer Rose"
+video_number: 1
+date: 2019-10-11
+video_id: 4uU9lZ-HSqA
+repository: Cabana_001_MaurerRose
```

Comment:
This looks like it is here accidentally - it's being overridden by line 8

Reply:
Thanks for pointing this out. I missed that it was already in there when I went to add the code. Should be fixed now.